source.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @module  web-audio-stream/source
+ *
+ * Source pull-stream for web-audio node
+ */
+'use strict';
+
+
+var pull = require('pull-stream/pull');
+var asyncMap = require('pull-stream/throughs/async-map');
+var infinite = require('pull-stream/sources/infinite');
+var createReader = require('./reader');
+
+module.exports = source;
+
+
+function source (node, options) {
+	let read = createReader(node, options);
+
+	//FIXME: too clumsy. I’d expect pull.asyncSource(read)
```

Comment:
@ahdinosaur is there a more graceful solution for async source?
[pull-async](http://npmjs.org/package/pull-async) is nice but seems to be called once.